src/NUnitFramework/tests/Assertions/CollectionAssertTest.cs

```diff
@@ -133,8 +133,8 @@ public void UniqueFailure_WithTwoNulls()
         [Test]
         public void AreEqual()
         {
-            var set1 = new SimpleObjectCollection("x", "y", "z");
-            var set2 = new SimpleObjectCollection("x", "y", "z");
+            var set1 = new SimpleEnumerable("x", "y", "z");
+            var set2 = new SimpleEnumerable("x", "y", "z");
 
             CollectionAssert.AreEqual(set1,set2);
```

Comment:
This test fails now, since Object.ReferenceEquals( set1, set2 ) == false, but CollectionAssert.AreEqual should be comparing the items, which are the same